pkg/statistics/handle/globalstats/global_stats_async.go

```diff
@@ -328,10 +425,22 @@ func (a *AsyncMergePartitionStats2GlobalStats) MergePartitionStats2GlobalStats(
 			metawg, _ := errgroup.WithContext(ctx)
 			mergeWg, _ := errgroup.WithContext(ctx)
 			metawg.Go(func() error {
-				return a.ioWorker(sctx, isIndex)
+				return a.loadFMSketchIOWorker(sctx, isIndex)
+			})
+			metawg.Go(func() error {
+				return a.loadCMSketchIOWorker(sctx, isIndex)
+			})
+			metawg.Go(func() error {
+				return a.loadHistogramAndTopNIOWorker(sctx, isIndex)
+			})
+			mergeWg.Go(func() error {
+				return a.dealFMSketchCPUWorker()
+			})
+			mergeWg.Go(func() error {
+				return a.dealCMSketchCPUWorker()
 			})
 			mergeWg.Go(func() error {
-				return a.cpuWorker(stmtCtx, sctx, opts, isIndex, tz, analyzeVersion)
+				return a.dealHistogramAndTopNCPUWorker(stmtCtx, sctx, opts, isIndex, tz, analyzeVersion)
```

Comment:
In history, we have found that concurrent loading is highly likely to consume a large amount of memory. If we cannot increase the concurrency of the processing workers, it is very likely to cause an OOM. I see that there is only concurrency in loading here, but not in calculation, which could lead to problems.